scripts/Units/armcroc.bos

```diff
@@ -130,6 +148,74 @@ AimFromPrimary(piecenum)
 	piecenum = turret;
 }
 
+
+
+
+
+
+
+
+
+AimSecondary(heading, pitch)
+{
+	signal SIG_AIM2;
+	set-signal-mask SIG_AIM2;
+	
+	move aturret to y-axis [15] speed [100];
+	move launcher to y-axis [4.5] speed [100];
+	turn launcher to x-axis <90.000000> speed <200.000000>;
+	
+	if( gun_2 == 1 )
+	{
+		turn asleeve to y-axis <0.000000> speed <400.000000>;
+	}
+	if( gun_2 == 2 )
+	{
+		turn asleeve to y-axis <120.027473> speed <400.000000>;
+	}
+	if( gun_2 == 3 )
+	{
+		turn asleeve to y-axis <-120.027473> speed <400.000000>;
+	}
+	turn aturret to y-axis <180> + heading speed <450.000000>;
+	total = <90.000000> + pitch;
+	if (total < 20000) {
+		total = <120>;
+	}
+	turn launcher to x-axis total speed <450.000000>;
+	wait-for-turn asleeve around z-axis;
+	wait-for-turn aturret around y-axis;
+	wait-for-turn launcher around x-axis;
+	start-script RestoreAfterDelay();
+	return (1);
+}
+
+FireSecondary()
+{
+	gun_2 = aflare;//why would it ever not be this?
```

Comment:
probably because of the ++ in the next line

Reply:
I specifically hard-set it there because there was no godly reason to increment it infinitely. Initially it seemed to loop around a flare + next 2 parts, which would make (old) sense if there -were- 3 flares in a row, but instead it hopped about a few bits of the head.  Removed it now anyway as there really is no need.